### DISCUSSION TITLE
Set like button for combination setter number?

### DISCUSSION THREAD

**Discussion by Fl0r14n** [GrandOrgue/grandorgue#1236] [Developers]
I am wondering if there is a possibility of a button that will initiate a combination setter change state that will wait for 3 digits.
Currently to set combination 130, I have to press +100,  3 x +10 and _0. What I would like is a button (let's call it N) that when pressed will wait for me to add _1,  then _3, then _0 to jump to combination 130. For 024 will be N + _0 + _2 + _4

The existing functionality, for when I am on 130 and want to jump to 133, in which I just press _3, is great.

If I press N then press just 2 digits let's say, it will stay in a suspended state until I press the last digit. This is useful for when during a play, I have to switch from 130 back to let's say 126, without pressing 4 x < which creates side effects with registration (I know it's an edge case and why not duplicate the registration to 131 and just do > the whole piece)

That will help me a lot with my organ where I have 0 - 9 < > T C under the first manual, so that I don't use the keyboard

**Comment by oleg68** [comment]
@Fl0r14n Thank you very much for a good idea of enchancement.
